src/yb/master/master-path-handlers.cc

```diff
@@ -311,7 +325,8 @@ void MasterPathHandlers::TServerDisplay(const std::string& current_uuid,
                                         std::vector<std::shared_ptr<TSDescriptor>>* descs,
                                         TabletCountMap* tablet_map,
                                         std::stringstream* output,
-                                        const int hide_dead_node_threshold_mins) {
+                                        const int hide_dead_node_threshold_mins,
+                                        bool tserver_clocks_view) {
```

Comment:
It would be better to use an enum for this purpose.
Or even a more abstract way with class/function that renders a single server view.